src/test/java/seedu/edulog/logic/commands/EditCommandTest.java

```diff
@@ -1,40 +1,40 @@
-import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.logic.commands.CommandTestUtil.showStudentAtIndex;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_STUDENT;
-import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_STUDENT;
-import static seedu.address.testutil.TypicalStudents.getTypicalAddressBook;
+import static seedu.edulog.logic.commands.CommandTestUtil.DESC_AMY;
+import static seedu.edulog.logic.commands.CommandTestUtil.DESC_BOB;
+import static seedu.edulog.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.edulog.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static seedu.edulog.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.edulog.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.edulog.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.edulog.logic.commands.CommandTestUtil.showStudentAtIndex;
+import static seedu.edulog.testutil.TypicalIndexes.INDEX_FIRST_STUDENT;
+import static seedu.edulog.testutil.TypicalIndexes.INDEX_SECOND_STUDENT;
+import static seedu.edulog.testutil.TypicalStudents.getTypicalEduLog;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.commons.core.index.Index;
-import seedu.address.logic.Messages;
-import seedu.address.logic.commands.EditCommand.EditStudentDescriptor;
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.UserPrefs;
-import seedu.address.model.calendar.EdulogCalendar;
-import seedu.address.model.student.Student;
-import seedu.address.testutil.EditStudentDescriptorBuilder;
-import seedu.address.testutil.StudentBuilder;
+import seedu.edulog.commons.core.index.Index;
+import seedu.edulog.logic.Messages;
+import seedu.edulog.logic.commands.EditCommand.EditStudentDescriptor;
+import seedu.edulog.model.EduLog;
+import seedu.edulog.model.Model;
+import seedu.edulog.model.ModelManager;
+import seedu.edulog.model.UserPrefs;
+import seedu.edulog.model.calendar.EdulogCalendar;
+import seedu.edulog.model.student.Student;
+import seedu.edulog.testutil.EditStudentDescriptorBuilder;
+import seedu.edulog.testutil.StudentBuilder;
 
 /**
  * Contains integration tests (interaction with the Model) and unit tests for EditCommand.
  */
 public class EditCommandTest {
 
-    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), new EdulogCalendar());
+    private Model model = new ModelManager(getTypicalEduLog(), new UserPrefs(), new EdulogCalendar());
```

Comment:
will take note of this 2-model approach, with both the student list and the calendar.